CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.0-beta.18 (2017-04-21)
+
+- Allowing `skyux test` to not fail if no `*.spec.ts` files are specified.
```

Comment:
I like your version better, feel free to push it to this branch, if you don't mind.